### PR TITLE
RS Cheatsheet

### PR DESCRIPTION
**Pull request by ldunnplaymint**
Trying to consolidate everything a user will need to work against RS

### CODE REVIEW

**Comment by ldunnplaymint** [comment]
From Iain:
The connections bit needs work. The graphql bit should just refer to Redstone  graphql URL not local. Get rid of the "this is local" wanting and the link to the  original connections page 

And add a. bit about browsing to https://redstone.downstream.game/config.json to get game and NFt addresses in the composability section

File: tutorial/RSCHEATSHEET.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Redstone Cheatsheet
+  
+## How to get your own Zone!
+
+Head over to [redstone.downstream.game](redstone.downstream.game), connect your wallet and select the "Mint" button on the home page!
```
Comment (chrisfarms) [inline tutorial/RSCHEATSHEET.md:5]:
```suggestion
Head over to [redstone.downstream.game](https://redstone.downstream.game), connect your wallet and select the "Mint" button on the home page!
```

File: tutorial/RSCHEATSHEET.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+| `-k <private key>` | Sign with this private key (see warning below). |
+
+PLEASE NOTE: It is recommended not to pass a private key into the cli, if you do not use the -k flag, your default browser will open a new tab with a MetaMask auth flow.
+
+
+### Drawing a map
+
+<img src="images/tile-fabricator.png" width="200">
+
+If you navigate to [redstone.downstream.game/tile-fabricator](redstone.downstream.game/tile-fabricator), you will be able to draw a tile map using our Tile Fabricator! Once you are happy, you can hit the export button to get a yaml file which can be passed into the `ds-cli` and deployed against your Zone!
```
Comment (chrisfarms) [inline tutorial/RSCHEATSHEET.md:45]:
```suggestion
If you navigate to [redstone.downstream.game/tile-fabricator](https://redstone.downstream.game/tile-fabricator), you will be able to draw a tile map using our Tile Fabricator! Once you are happy, you can hit the export button to get a yaml file which can be passed into the `ds-cli` and deployed against your Zone!
```

File: tutorial/RSCHEATSHEET.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+### Building Fabricator
+<img src="images/building-fabricator.png" width="200">
+
+Browse to [redstone.downstream.game/building-fabricator](redstone.downstream.game/building-fabricator)
```
Comment (chrisfarms) [inline tutorial/RSCHEATSHEET.md:50]:
```suggestion
Browse to [redstone.downstream.game/building-fabricator](https://redstone.downstream.game/building-fabricator)
```

File: tutorial/RSCHEATSHEET.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+The Building Fabricator allows you to deploy buildings and items to Downstream without having to leave the game! Once configured hit the ___Deploy___ button.
+
+To add new behaviour to the building, once configured, hit the ___Export___ button and follow the instructions below for deploying any code changes from the command line.
+
+You can also use the __building-fabricator__ to _export_ building sources and apply it using the cli to Downstream:
+```ds apply -n redstone -z <zone-number> -f ./BasicFactory.yaml```
```
Comment (chrisfarms) [inline tutorial/RSCHEATSHEET.md:57]: code block starts (and ends)  '```' should be on their own lines ... if you wanted inline just use '`'

**Review by chrisfarms — APPROVED** [review]
LGTM ... few markdown syntaxy bits

File: tutorial/RSCHEATSHEET.md (hunk 1 above)
Comment (5p0rt5BEArD) [inline tutorial/RSCHEATSHEET.md:5]: link does not work - need https:// I think

File: tutorial/RSCHEATSHEET.md (hunk 2 above)
Comment (5p0rt5BEArD) [inline tutorial/RSCHEATSHEET.md:45]: link does not work - need https:// I think

File: tutorial/RSCHEATSHEET.md (hunk 3 above)
Comment (5p0rt5BEArD) [inline tutorial/RSCHEATSHEET.md:50]: link does not work - need https:// I think

**Review by 5p0rt5BEArD — CHANGES_REQUESTED** [review]
all looks good
couple of links broken

**Review by JackPlaymint — APPROVED** [review]

**Pull request merged**